apps/buildog/src/app/organizations/[organizationId]/layout.tsx

```diff
@@ -21,16 +21,16 @@ export default function Layout({
 }) {
   const { organizationId } = params;
 
+  const organizations = await getOrganizations();
```

Comment:
getOrganizations function is not return promise we don't need to await here. When backend complete the development of organizations you can call the backend in useEffect.